packages/cli/src/commands/dev/index.ts

```diff
@@ -74,11 +74,14 @@ export default async function main(client: Client) {
     argv = getArgs(client.argv.slice(2), {
       '--listen': String,
       '-l': '--listen',
-      '--confirm': Boolean,
+      '--yes': Boolean,
+      '-y': '--yes',
 
       // Deprecated
       '--port': Number,
       '-p': '--port',
+      '-c': '--yes',
```

Comment:
Let's add a deprecation warning for when `--confirm` is used?

Reply:
I asked that exact question on the Linear card (https://linear.app/vercel/issue/VCCLI-226#comment-65e30f77) and Sean was concerned about writing a warning to `stdout` for the `deploy` command. I'd be certainly happy to add a warning to all or some of the commands. Let me know what you think.

Reply:
I saw that comment and I thought that I followed-up but I guess I didn't hit send. We write to `stderr` (via `client.output`) for logging messages so that shouldn't be a concern 😃

Reply:
Do you mean `client.warn()` like this: https://github.com/vercel/vercel/blob/main/packages/cli/src/commands/dev/index.ts#L86?

Reply:
Fixed! :)